doc/mediation/README.md

```diff
@@ -0,0 +1,87 @@
+[TOP](../#top)　>　Ad Mediation
+
+---
+
+# Ad Mediation
+
+Ad mediation is a digital advertising strategy where multiple ad networks are managed through a single SDK to help publishers increase advertising revenue, fill rates, and efficiency for their available ad space.
+
+We currently support the ad banner feature of GoogAd Mediation SDK.
+
+---
+
+## How to use
+
+### Prepare Google Mobile Ads SDK
+
+Please follow the [introduction](https://developers.google.com/admob/android/banner) to get your ad unit ID and load ads from google.
+
+#### Enable mediation configurations in Admob console.
+
+Ask RUNA support team to enable the configuration of the Admob Mediation. The following 2 steps will be done.
```

Comment:
Who operates the Admob account is responsible for these steps.
RUNA support team may not have the authority to do these.

Reply:
@wei-b-wu-rakuten 
Will business side members in each team set up and manage admob configurations?